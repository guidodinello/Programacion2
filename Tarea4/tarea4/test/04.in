# Implementar operaciones de pila
estaVaciaPila
apilar 1
apilar 2
apilar 3
apilar 4
apilar 5
apilar 6
apilar 7
apilar 8
apilar 9
estaVaciaPila
estaLlenaPila
apilar 10
estaVaciaPila
estaLlenaPila
cima
desapilar 
desapilar
cima
desapilar 
desapilar
cima
desapilar 
desapilar
cima
desapilar 
desapilar
cima
desapilar 
cima
desapilar
estaVaciaPila
estaLlenaPila
Fin
