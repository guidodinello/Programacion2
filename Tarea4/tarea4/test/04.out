1>#  Implementar operaciones de pila.
2>p está vacia.
3>Apilado.
4>Apilado.
5>Apilado.
6>Apilado.
7>Apilado.
8>Apilado.
9>Apilado.
10>Apilado.
11>Apilado.
12>p NO está vacia.
13>p NO está llena.
14>Apilado.
15>p NO está vacia.
16>p está llena.
17>10.
18>Desapilado.
19>Desapilado.
20>8.
21>Desapilado.
22>Desapilado.
23>6.
24>Desapilado.
25>Desapilado.
26>4.
27>Desapilado.
28>Desapilado.
29>2.
30>Desapilado.
31>1.
32>Desapilado.
33>p está vacia.
34>p NO está llena.
35>Fin.
